### DISCUSSION TITLE
Troubleshooting Initial Page Load

### DISCUSSION THREAD

**Discussion by hahp97** [vercel/community#3461] [Help]
### Summary

Hi, I am facing an issue with the initial page load (white screen) on my Next.js web project. I tried implementing a loading fallback similar to how it's done in React.js, but all the pages on the website I'm currently working on still show a white screen. Is there any way to troubleshoot and resolve this in Next.js? My background is in app development, and I previously worked with React Native. Recently, I started learning Next.js for web development, but I'm not very familiar with it yet [https://travelnest.space/](https://travelnest.space/)

And the initial loading page like on this website, what are the keywords to search for it, everyone? https://skymavis.com/

Here is my source code: [https://github.com/hahp97/uwuwNext](https://github.com/hahp97/uwuwNext)
Thank you all.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/48684732/8cea0ff3-5267-47c0-a35c-8cc94d41387c)
![image](https://github.com/vercel/community/assets/48684732/1f90fba6-0726-43ff-b315-6dabd945031c)

**Comment by amyegan** [comment]
Hey @hahp97. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give us a better understanding of your code setup so we can work together and debug this.

**Reply by hahp97** [reply]
Thank you, here is my source code: [https://github.com/hahp97/uwuwNext]( https://github.com/hahp97/uwuwNext)
I have tried to research the above problems but I don't seem to understand the optimal mechanism for the first page load in nextjs.
